Makefile

```diff
@@ -164,3 +165,67 @@ ifneq ($(HAS_WEBAPP),)
 	rm -fr webapp/node_modules
 endif
 	rm -fr build/bin/
+
+# debug builds and deploys a debug version of the webapp, for only my architecture.
+.PHONY: debug
+debug: stop
+	./build/bin/manifest apply
+ifneq ($(HAS_SERVER),)
+	mkdir -p server/dist
+	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build -gcflags "all=-N -l" -o dist/plugin-darwin-amd64
```

Comment:
we should use set this up for a local arch, not hardcoded; otherwise I'd suggest OS-specific targets, maybe `debug-osx`, etc.

Reply:
I'm now just requiring an OS env variable (I'll put it in the instructions). Personally I like that better than typing more every time I want to reload (3/5). Or do you prefer `make debug-OSX`?